src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -258,6 +264,25 @@ private InitCodeView buildInitCodeViewRequestObject(
     return paths.stream().map(path -> root.subTree(path)).collect(ImmutableList.toImmutableList());
   }
 
+  @VisibleForTesting
+  static void assertNoOverlap(InitCodeNode root, List<String> paths) {
+    HashMap<InitCodeNode, String> nodes = new HashMap<>();
```

Comment:
Explain the map key/values: `nodes[node]=path`

Reply:
Done